go/vt/vtorc/process/health.go

```diff
@@ -73,6 +74,7 @@ type HealthStatus struct {
 	Hostname           string
 	Token              string
 	IsActiveNode       bool
+	HasDiscoveredOnce  bool
```

Comment:
I believe golang convention is to omit verbs like `Is` and `Has`, but I see we already have an `IsActiveNode`.

Reply:
Fixed.